src/app/components/Toolbar/Features/LayerContainer/index.tsx

```diff
@@ -23,11 +23,9 @@ export const LayerContainer = ({ animation, children, hideLayer }: LayerContaine
       modal
       position="top"
       margin={isMobile ? 'none' : 'xlarge'}
+      style={{ width: '100%', maxWidth: isMobile ? 'none' : '700px' }}
```

Comment:
Note: this only fixes horizontal scrollbar part of https://github.com/oasisprotocol/oasis-wallet-web/issues/1776

Reply:
I'd rename PR and commit and changelog to specify horizontal